app/src/main/res/layout/activity_user_profile.xml

```diff
@@ -99,6 +99,7 @@
                 android:textSize="24sp"
                 android:textColor="@android:color/white"
                 android:text="1234567"
+                android:onClick="callPhone"
                 android:focusable="false"
                 android:clickable="false"
                 android:background="@null"
```

Comment:
Also I have no idea how hard this would be and it is maybe an enhancement (could just make an enhancement issue for it) but the call/email action doesn't take place right away for me so some kind of visual clue that I pressed something clickable would be nice. Like you know when a button flashes or something? If someone knows how to do this now that would be a cool addition, but I don't think it's necessary per se.